File: src/main/java/tech/zolhungaj/amqapi/servercommands/objects/AvatarDriveNomination.java
package tech.zolhungaj.amqapi.servercommands.objects;

import com.squareup.moshi.Json;

public record AvatarDriveNomination(
    @Json(name = "name")
    String avatarName,
    @Json(name = "value")
    double amount
) {
}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/objects/AvatarDriveNomination.kt
package tech.zolhungaj.amqapi.servercommands.objects

import com.squareup.moshi.Json

@JvmRecord
data class AvatarDriveNomination(
    @Json(name = "name") val avatarName: String,
    @Json(name = "value") val amount: Double
) 